rubygem-foreman_templates/rubygem-foreman_templates.spec

```diff
@@ -35,6 +35,10 @@ Requires: %{?scl_prefix}rubygem(git)
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
 BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildRequires: %{?scl_prefix_ruby}rubygems
+BuildRequires: %{?scl_prefix}rubygem(diffy)
+BuildRequires: %{?scl_prefix}rubygem(git)
```

Comment:
that should probably be listed at https://github.com/theforeman/foreman-packaging/blob/rpm/develop/rubygem-git/rubygem-git.spec#L15 as a runtime dependency too, should I send a separate patch for it?